test/integration/api/integrations_controller_test.rb

```diff
@@ -170,4 +170,15 @@ def test_edit
     get edit_admin_service_integration_path(service_id: service.id)
     assert_response :not_found
   end
+
+  def
+    service = FactoryBot.create(:simple_service, account: @provider)
+    get admin_service_integration_path(service_id: service)
+    service.stubs(:oauth?).retuns(true)
+    assert_response :success
+    assert_match  "Example curl for testing", response.body
+    service.stubs(:oauth?).retuns(false)
```

Comment:
Either you put this 2nd test in a different method or at least please add a space between them.